CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKRenderer.java

```diff
@@ -75,51 +76,52 @@ Favour full names over abbreviations (eg Initialise() over Init()).
 
 TODO: this is still not a good reason.  Maybe just stylistic so resource types are differentiated from other CVK types?
 Resource types should expose a Create factory method and have private constructors so that we don't have to worry
-about having resource types in partially initialised states.
+about having resource types in partially initialised states.  We can also indicate failure by returning null from Create
+rather than throwing exceptions from ctors.
 
 */
 
 public class CVKRenderer implements ComponentListener {
     
-    private int currentFrame = 0;
-    private CVKInstance cvkInstance = null;
-    private CVKDevice cvkDevice = null;
+    private final CVKCanvas cvkCanvas;  
```

Comment:
canvas is the parent right? I wonder if it is worth making that clear here, even by a comment

Reply:
Done